frontend/src/components/elements/DeckGlChart/DeckGlChart.tsx

```diff
@@ -64,6 +64,8 @@ interface ViewPort extends ViewState {
 }
 
 export class DeckGlChart extends PureComponent<PropsWithHeight, State> {
+  static readonly deltaType = "deck_gl_chart"
```

Comment:
Add a comment explaining what this is for.

(And actually, if you used your other solution, this would be clearer... So now I'm thinking that solution is definitely better!)